From 6fbcba24ff8668a671cf9ef16bd8cf8673753c2e Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 31 Mar 2022 20:19:29 +0300
Subject: [PATCH] [SW-77251] common: move round up macro to header

This patch moves round up macro to common header file as it required
by other files.

Change-Id: Id275d1b1c6e2db4f3f4ae50dcc77607128c061a5
---

diff --git a/common/boot_info.c b/common/boot_info.c
index 309af4b..f46bb1a 100644
--- a/common/boot_info.c
+++ b/common/boot_info.c
@@ -14,7 +14,6 @@
 #include "boot_info.h"
 
 #define BOOT_INFO_BASE		(HL_GEN_MEM_BASE + HL_SCRATCHPAD_SRAM_OFFSET)
-#define ROUND_UP4(n)		(((n) + sizeof(uint32_t) - 1) & ~(sizeof(uint32_t) - 1))
 
 static struct boot_info boot_info = { .ptrs = { .count = 0 } };
 
diff --git a/include/arcboot.h b/include/arcboot.h
index 09c72d2..8eabe5f 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -32,6 +32,7 @@
 #define __unused	__attribute__((__unused__))
 #endif
 
+#define ROUND_UP4(n)			(((n) + sizeof(uint32_t) - 1) & ~(sizeof(uint32_t) - 1))
 #define BIT(nr)					(1 << (nr))
 #define ARRAY_SIZE(x)				(sizeof(x) / sizeof((x)[0]))
 
